index.html

```diff
@@ -317,6 +319,73 @@ <h2>Nike Air Max</h2>
+        alt="">
+    <img class="avatar round md"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar round lg"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar sm"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar md"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar lg"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+
+    <!-- BADGES -->
+    <div class="badge-conatiner">
+        <div class="badge">
+            <span class="material-icons-outlined">
+                notifications
+            </span>
+            <sup class="badge-icon">4</sup>
+        </div>
+        <div class="badge">
+            <span class="material-icons-outlined">
+                shopping_bag
+            </span>
+            <sup class="badge-icon">4</sup>
+        </div>
+        <div class="badge">
+            <span class="material-icons-outlined">
+                mail
+            </span>
+            <sup class="badge-icon">4</sup>
+        </div>
+    </div>
+
+    <div class="status">
+        <img class="avatar round sm"
+            src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+            alt="">
+        <div class="online"></div>
+    </div>
+    <div class="status">
+        <img class="avatar round sm"
+            src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+            alt="">
+        <div class="away"></div>
+    </div>
+    <div class="status">
+        <img class="avatar round sm"
+            src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+            alt="">
+        <div class="dnd"></div>
+    </div>
+    <div class="status">
+        <img class="avatar round sm"
+            src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+            alt="">
+        <div class="offline"></div>

```

Comment:
I am sure these empty divs are there for some reason!

Reply:
yes, it is there to create a status for avatars.